src/Polly.Shared/Duration/ISleepDurationStrategy.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace Polly.Duration
+{
+    /// <summary>
+    /// An interface defining strategies for managing the sleep-duration of retries. 
+    /// </summary>
+    public interface ISleepDurationStrategy
```

Comment:
Happy to discuss a better name if necessary

Reply:
I am now debating that, in the absence of the integration overloads, whether we need to make this interface `public` anymore. If we made it `internal` it would be less api noise. But in that case consumers would not be able to write their own extensions over any concrete class. For example, a consumer would not be able to write a `Continuous` extension like I had previously.
So maybe it's good to keep the interface public?